domain/src/main/java/tachiyomi/domain/manga/interactor/FetchInterval.kt

```diff
@@ -107,7 +107,7 @@ class FetchInterval(
             interval.absoluteValue.takeIf { interval < 0 }
                 ?: increaseInterval(interval, timeSinceLatest, increaseWhenOver = 10),
         )
-        return latestDate.plusDays((cycle + 1) * interval.toLong()).toEpochSecond(dateTime.offset) * 1000
+        return latestDate.plusDays((cycle + 1) * interval.absoluteValue.toLong()).toEpochSecond(dateTime.offset) * 1000
```

Comment:
Should just fix this inside `calculateInterval`

Reply:
`calculateInterval` is used to automatically calculate the interval based on chapter's release date, when SmartUpdate is set to 'default'.

The case here is when user set a custom SmartUpdate, interval is set to a negative value and used as is, without calling to calculateInterval.